yt_dlp/downloader/external.py

```diff
@@ -331,7 +335,6 @@ def _make_cmd(self, tmpfilename, info_dict):
             cmd += ['--dir', self._aria2c_filename(dn) + os.path.sep]
         if 'fragments' not in info_dict:
             cmd += ['--out', self._aria2c_filename(os.path.basename(tmpfilename))]
-        cmd += ['--auto-file-renaming=false']
```

Comment:
This is intentionally kept after `_configuration_args` so that user cannot override it

Reply:
Alright. I've adjusted this and the other flags a user should not override.